config-service-api/src/main/proto/org/hypertrace/config/service/v1/config_service.proto

```diff
@@ -68,7 +68,7 @@ message GetAllConfigsRequest {
 }
 
 message GetAllConfigsResponse {
-  // list of config values along with the associated contexts
+  // list of config values along with the associated contexts, sorted in the ascending order of their creation time
```

Comment:
I think usually the client would want the configs to be returned in the order of creation time. So, added this extra condition.

Reply:
That makes sense. If we need to add sort support later, we can